projects/plugins/jetpack/modules/search/instant-search/components/search-app.jsx

```diff
@@ -159,11 +159,24 @@ class SearchApp extends Component {
 
 	handleInput = debounce( event => {
 		// Reference: https://rawgit.com/w3c/input-events/v1/index.html#interface-InputEvent-Attributes
-		if ( event.inputType.includes( 'format' ) || event.target.value === '' ) {
+		// NOTE: inputType is not compatible with IE11, so we use optional chaining here. https://caniuse.com/mdn-api_inputevent_inputtype
+		if ( event.inputType?.includes( 'format' ) ) {
+			return;
+		}
+
+		// If user presses enter, propagate the query value and immediately show the results.
+		if ( event.key === 'Enter' ) {
```

Comment:
This doesn't work for me in FF or Chrome. I don't see a `key` property on the InputEvent:

<img width="1049" alt="Screen Shot 2021-01-28 at 14 15 09" src="https://user-images.githubusercontent.com/17325/106075925-75c65e80-6173-11eb-843a-48212f5b5fa0.png">

Reply:
Good catch. Fixed this by creating a separate keydown handler and moving the necessary code over.